uml/state_diagrams/power_management.plantuml

```diff
@@ -0,0 +1,24 @@
+@startuml
+
+skinparam shadowing false
+
+title Power Management State Diagram
+
+state "Full Power" as Fp : entry: Set Display Brightness to Stored State\nentry: Set Bluetooth to Stored State\nentry: Set Health Notification to Stored State
+state "Low Power" as Lp : entry: Trigger Motor Vibration for 2 secs\nentry: Set Display Brightness State = Low\nentry: Disable Bluetooth\nentry: Disable Health Notification 
+state "Power Down" as Pd : entry: Turn Off Device
+
+state Choice <<choice>>
+
+[*] --> Choice
+Choice --> Lp : [Battery SoC <= 10% &&\nBattery SoC > 2%]
```

Comment:
For compatibility with the sequence diagram, you can add the case of Battery not available here

Reply:
I've pushed a new version - do you think it is sufficient ?

Reply:
ok